conf/init/01_validate_config.sh

```diff
@@ -0,0 +1,6 @@
+#! /bin/sh
+QUAYPATH=${QUAYPATH:-"."}
+QUAYCONF=${QUAYCONF:-"$QUAYPATH/conf"}
+
+echo "Validating Configuration"
+config-tool validate -c $QUAYCONF/stack/ --mode online
```

Comment:
add env var to skip check

Reply:
For now, let's make this a non-failing check for testing purposes.

Perhaps also change the order of this to after the certs_create.sh and certs_install.sh are run.